daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/InsertOrdMap.scala

```diff
@@ -45,6 +49,6 @@ object InsertOrdMap {
   def empty[K, V]: InsertOrdMap[K, V] = Empty.asInstanceOf[InsertOrdMap[K, V]]
 
   def apply[K, V](entries: (K, V)*): InsertOrdMap[K, V] =
-    new InsertOrdMap(entries.map(_._1)(breakOut), HashMap(entries: _*))
+    (empty[K, V] /: entries)(_ + _)
```

Comment:
`/:` is deprecated in Scala 2.12.10 and 2.13; let's use `foldLeft` instead. 🙂

Reply:
fixed